OrbitGl/App.cpp

```diff
@@ -854,6 +854,12 @@ void OrbitApp::SendToUiNow(const std::string& message) {
   }
 }
 
+//-----------------------------------------------------------------------------
+void OrbitApp::SendInfoToUi(const std::string& title, const std::string& text) {
+  std::string message = "info:" + title + "\n" + text;
+  SendToUiNow(message);
+}
```

Comment:
looks like this should not be here - rebase went wrong?

Reply:
As @dpallotti said. This is for an information box, the one below is for an error box.